apps/web/app/[locale]/(with-layout)/my-page/edit-profile/page.tsx

```
@@ -1,56 +1,175 @@
+          })
+          .catch((error) => {
+            console.error('프로필 이미지 로드 실패:', error);
+            setProfileImage(null);
+          });
+      }
+    }
+  }, [profileQuery.data, form]);
+
+  const handleCheckAvailability = async () => {
+    const checkIdMutation = useCheckIdAvailability();
+    const id = form.getValues('id');
```

Biome (2.1.2)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)